seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/CoordinatorService.java

```diff
@@ -442,6 +442,7 @@ public PassiveCompletableFuture<Void> submitJob(long jobId, Data jobImmutableInf
                         runningJobInfoIMap.remove(jobId);
                         runningJobMasterMap.remove(jobId);
                     }
+                    jobMaster.setClassLoader(null);
```

Comment:
I'm not sure if this method is reasonable, because `JobMaster` will be cleaned up from `runningJobMasterMap` when the job ends, which means that `JobMaster` will not be referenced after that, and his member variable `classloader` will not be referenced, so then It can be recycled normally. Can you tell me the deeper reason for this change?

Reply:
> I'm not sure if this method is reasonable, because `JobMaster` will be cleaned up from `runningJobMasterMap` when the job ends, which means that `JobMaster` will not be referenced after that, and his member variable `classloader` will not be referenced, so then It can be recycled normally. Can you tell me the deeper reason for this change?

Yes, gc does not recycle the classLoader